src/assets/styles/main.css

```
@@ -0,0 +1,247 @@
+.container {
+  flex: 0.93;
+  padding: 25px 0 31px 0;
+}
+
+.search-bar {
+  gap: 5px;
+  background-color: white;
+  padding: 5px 0;
```

it's better add the measure units to be specific

```suggestion
  margin: 5px 0px;
```

---

Thanks you for the feedback, I omitted adding the units since I was not adding any amount, but adding it is always better